base_multi_image/models/image.py

```diff
@@ -174,3 +188,9 @@ def _check_store(self):
         if self.storage == 'db' and not self.file_db_store:
             raise exceptions.ValidationError(
                 'You must provide an attached file for the image.')
+
+    @api.constrains('storage', 'attachment_id')
+    def _check_store(self):
+        if self.storage == 'filestore' and not self.attachment_id:
+            raise exceptions.ValidationError(
```

Comment:
`base_multi_image/models/image.py:195: [C8107(translation-required), Image._check_store] String parameter of raise "ValidationError" requires translation. Use _('You must provide an attachment for the image.')`